mappings/net/minecraft/item/impl/ItemFish.mapping

```diff
@@ -1,5 +1,35 @@
 CLASS none/aeh net/minecraft/item/impl/ItemFish
+	CLASS none/aeh$a FishType
+		FIELD a COD Lnone/aeh$a;
+		FIELD b SALMON Lnone/aeh$a;
+		FIELD c CLOWNFISH Lnone/aeh$a;
+		FIELD d PUFFERFISH Lnone/aeh$a;
+		FIELD e TYPES Ljava/util/Map;
+		FIELD f id I
+		FIELD g name Ljava/lang/String;
+		FIELD h rawHunger I
+		FIELD i rawSaturation F
+		FIELD j cookedHunger I
+		FIELD k cookedSaturation F
+		FIELD l cookable Z
+		METHOD <init> (Ljava/lang/String;IILjava/lang/String;IF)V
+		METHOD <init> (Ljava/lang/String;IILjava/lang/String;IFIF)V
+		METHOD a getId ()I
+		METHOD a getTypeFromId (I)Lnone/aeh$a;
+			ARG 0 id
+		METHOD a getTypeFromStack (Lnone/aeq;)Lnone/aeh$a;
+			ARG 0 itemStack
+		METHOD b getName ()Ljava/lang/String;
+		METHOD c getRawHunger ()I
+		METHOD d getRawSaturation ()F
+		METHOD e getCookedHunger ()I
+		METHOD f getCookedSaturation ()F
+		METHOD g canCook ()Z
```

Comment:
I'd prefer `isCookable`, and in general use `isXYZ` for every similar simple method returning a bool (`cookable` would be used for fields?). `canXYZ` should be used for method that take more parameters, like for example `canEat(ItemStack, EntityPlayer)`.